### PR TITLE
DOCS-260 migrate table components to markdown tables

### PR DESCRIPTION
**Pull request by alexisintech**
This PR serves to remove `<Table />` components where they aren't necessary, in favor of markdown tables. This instills a practice that should be used moving forward in the Docs repo, and will be talked about in the upcoming Style Guide.

This PR also removes unnecessary imports, which are just tech debt.

### CODE REVIEW

File: docs/components/clerk-provider.mdx [hunk 1]
```diff
@@ -93,14 +93,13 @@ ReactDOM.render(
...
 | `clerkJSVersion?`         | `string`                                                                              | Define the npm version for `@clerk/clerk-js`.                                                                                                                                                                              |
 | `supportEmail?`           | `string`                                                                              | Optional support email for display in authentication screens. Will only affect [Clerk Components][components-ref] and not [Account Portal][ap-ref] pages.                                                                  |
-| `localization?`           | `object`                                                                              | Optional object to localize your Components. Will only affect Clerk Components and not Account Portal pages.                                                                                                               |
```
Comment (JacobMGEvans) [inline docs/components/clerk-provider.mdx:103]: Is this deprecated or removed?

**Review by JacobMGEvans — APPROVED** [review]

File: docs/components/clerk-provider.mdx (hunk 1 above)
Comment (royanger) [reply docs/components/clerk-provider.mdx:103]: > Is this deprecated or removed?

Were you have issues with this in one of the SDKs?

Comment (alexisintech) [reply docs/components/clerk-provider.mdx:103]: It was listed twice - it's now listed on line 105

Comment (JacobMGEvans) [reply docs/components/clerk-provider.mdx:103]: > It was listed twice - it's now listed on line 105

Awesome, thanks for the update.

**Pull request merged**